packages/aws-ses/README.md

```diff
@@ -0,0 +1,52 @@
+# @modelence/aws-ses
+
+AWS SES Adapter for modelence
+
+## Installation
+
+```bash
+npm i @modelence/aws-ses
+```
+
+## Overview
+
+This package provides `sendEmail` function that utilizes resend under the hood. The configuration can be set via Modelence Cloud or the following environment variables:
+
+- MODELENCE_EMAIL_SES_REGION
+- MODELENCE_EMAIL_SES_ACCESS_KEY_ID
+- MODELENCE_EMAIL_SES_SECRET_ACCESS_KEY
+
+## Simple usage
+
+```ts
+import { sendEmail } from '@modelence/resend';
+
+sendEmail({
+  to: 'test@example.com',
+  from: 'test@example.com',
+  subject: 'Test Email',
+  html: '<h1>Test Email</h1>',
+})
+```
+
+## Advanced example
+
+```tsx
+import { sendEmail } from '@modelence/resend';
+
+sendEmail({
+  to: 'test@example.com',
+  from: 'test@example.com',
+  subject: 'Test Email',
+  react: <h1>Test Email</h1>,
```

Comment:
Is it generally a standard practice to call this `react` instead of `body` (i.e. after the structure type)?